controllers/hostedcontrolplane/hostedcontrolplane_test.go

```diff
@@ -738,6 +739,11 @@ func newTestReconciler(t *testing.T, objs ...client.Object) *HostedControlPlaneR
 		t.Errorf("failed to add routev1 to scheme: %v", err)
 	}
 
+	err = avov1alpha2.AddToScheme(s)
+	if err != nil {
+		t.Errorf("unable to add avov1alpha2 scheme to test: %v", err)
+	}
+
```

Comment:
Looks like we add the new GVK to the scheme in the test file, but not in `main.go`. Does that need to happen in this PR as well?

I see the schemes used by these controllers defined [here](https://github.com/openshift/route-monitor-operator/blob/master/main.go#L61-L73) fwiw